sympy/combinatorics/coset_table.py

```diff
@@ -949,7 +948,7 @@ def modified_coincidence(self, alpha, beta, w):
         Parameters
         ==========
 
-        A coincident pair \alpha,\beta \in \Omega, w \in (Y∪Y^–1)
+        A coincident pair \alpha,\beta \in \Omega, w \in `Y \cup Y^{-1}`
```

Comment:
Shouldn't \alpha etc also be equations? (I know this is not what you have fixed though...)